src/Tribe/Views/V2/Assets.php

```diff
@@ -40,13 +40,29 @@ class Assets extends \tad_DI52_ServiceProvider {
 	public function register() {
 		$plugin = Plugin::instance();
 
+		tribe_asset(
+			$plugin,
+			'tribe-events-views-v2-skeleton',
+			'views-skeleton.css',
+			[
+				'tribe-common-skeleton-style',
+				'tribe-tooltipster-css',
+			],
+			'wp_enqueue_scripts',
+			[
+				'priority'     => 10,
+				'conditionals' => [ $this, 'should_enqueue_frontend' ],
+				'groups'       => [ static::$group_key ],
+			]
+		);
+
 		tribe_asset(
 			$plugin,
 			'tribe-events-views-v2-full',
 			'views-full.css',
 			[
-				'tribe-common-style',
-				'tribe-tooltipster-css',
+				'tribe-common-full-style',
```

Comment:
I assume we're working on the option for skeleton vs full in another ticket?

Reply:
yeah, that's a BE thing I'll have to connect with G on